tsconfig.json

```diff
@@ -14,6 +14,7 @@
     ],
     "experimentalDecorators": true,
     "emitDecoratorMetadata": true,
+    "esModuleInterop": true,
```

Comment:
Why is this change needed?

Reply:
It's because of limitations in the import of modules where `exports = ...` syntax was used. You either get this error (for the default import)
![image](https://user-images.githubusercontent.com/7657388/111884761-8cdb3d00-89bb-11eb-96f3-e8b7f6e2eb8d.png)
Or you get this one (for the `*` variant)
![image](https://user-images.githubusercontent.com/7657388/111884825-f3605b00-89bb-11eb-907b-4c86fea6dc68.png)
In either case, the solution is the same, which is to turn on the `esModuleInterop` flag. With `tslib` being now referenced in most TS projects, there's little overhead to it anyway.

Reply:
I see, thanks :+1: